www/artworks/edit.php

```diff
@@ -0,0 +1,73 @@
+<?
+use function Safe\session_unset;
+
+session_start();
+
+$exception = $_SESSION['exception'] ?? null;
+
+try{
+	$artwork = Artwork::GetByUrl(HttpInput::Str(GET, 'artist-url-name') ?? '', HttpInput::Str(GET, 'artwork-url-name') ?? '');
```

Comment:
We have to also pull the artwork from the session, not just the URL. If the user submitted edits but they were rejected, the form must show they changes they made, not the original artwork. This is the same logic in the new artwork form, you can basically copy and paste that logic into here for the most part.

Reply:
Yep, done.